ecoscope/plotting/plot.py

```diff
@@ -158,42 +150,6 @@ def add_seasons(fig, season_df):
     return fig
 
 
-def collar_event_timeline(relocations, collar_events):
```

Comment:
where has this function moved to?

Reply:
I have moved the function to the Collared Subject Report notebook, in the collar event cell. I have tested it to make sure it runs just fine.